Mayhemfile

```diff
@@ -0,0 +1,5 @@
+project: libriscv 
+target: fuzzme_2
```

Comment:
I would rename this target to `vmfuzzer32`, as that's the name of the actual target